src/main/java/net/minestom/server/entity/Entity.java

```diff
@@ -1609,8 +1610,20 @@ public void askSynchronization() {
      * Set custom cooldown for position synchronization.
      *
      * @param cooldown custom cooldown for position synchronization.
+     * @deprecated Replaced by {@link #setCustomSynchronizationCooldown(Duration)}
      */
-    public void setCustomSynchronizationCooldown(@Nullable UpdateOption cooldown) {
+    @SuppressWarnings("removal")
+    @Deprecated(forRemoval = true)
+    public void setCustomSynchronizationCooldown(@Nullable net.minestom.server.utils.time.UpdateOption cooldown) {
```

Comment:
There's so many breaking changes in this PR atm. I think we should either break them all or try and not break anything - no need for half measures.

Reply:
Correct me if i'm wrong, but this pr won't break a single project (exept of those which rely on `TimeUnit.toMilliseconds`).